src/mediapipe_internal/mediapipegraphconfig.cpp

```diff
@@ -31,6 +31,7 @@ namespace ovms {
 
 const std::string DEFAULT_GRAPH_FILENAME = "graph.pbtxt";
 const std::string DEFAULT_SUBCONFIG_FILENAME = "subconfig.json";
+const std::string DEFAULT_MODELMESH_SUBCONFIG_FILENAME = "config.json";
```

Comment:
this name is not limited to modelmesh only.
I would call it DEFAULT_SUBCONFIG_FILENAME2 or we could make is a list of 2 elements